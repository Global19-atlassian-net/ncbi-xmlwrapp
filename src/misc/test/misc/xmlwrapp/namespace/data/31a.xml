<?xml version="1.0"?>
<root xmlns:foo="http://some.name.space" xmlns="default">
    <nested>
        <foo:deep>
        </foo:deep>
    </nested>
</root>
