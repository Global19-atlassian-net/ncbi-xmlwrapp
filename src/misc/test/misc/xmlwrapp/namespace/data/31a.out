<?xml version="1.0"?>
<root xmlns:foo="http://some.name.space" xmlns="default">
    <nested>
        <deep>
        </deep>
    </nested>
</root>
